packages/editor-ui/src/stores/workflowHistory.store.ts

```diff
@@ -0,0 +1,47 @@
+import { ref } from 'vue';
+import { defineStore } from 'pinia';
+import * as whApi from '@/api/workflowHistory';
+import { useRootStore } from '@/stores/n8nRoot.store';
+import type { WorkflowHistory, WorkflowVersion } from '@/types/workflowHistory';
+
+export const useWorkflowHistoryStore = defineStore('workflowHistory', () => {
+	const rootStore = useRootStore();
+
+	const workflowHistory = ref<WorkflowHistory[]>([]);
+	const workflowVersion = ref<WorkflowVersion | null>(null);
+
+	const getWorkflowHistory = async (
+		workflowId: string,
+		queryParams: Parameters<typeof whApi.getWorkflowHistory>[2],
+	): Promise<WorkflowHistory[]> =>
```

Comment:
Let's remove inferrable return types. Less clutter without losing any info. Same elsewhere.

```suggestion
	const getWorkflowHistory = async (
		workflowId: string,
		queryParams: Parameters<typeof whApi.getWorkflowHistory>[2],
	) =>
```